### ISSUE TITLE
Notification mail without content a bit useless

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/activity#287]
I got this mail:
```
There was some activity at Nextcloud
•Today at 12:17 PM
Roeland posted an announcement
``` 

But I did not received the content of the announcement, which then requires an additional step (login, click on announcement).

**Comment by nickvergessen** [comment]
Yes, it used to have the title in there too, not sure what happened.

**Comment by tobiasKaminsky** [comment]
I would even go further and say that it should include all content.
Or do you expect this to be a security problem?

**Comment by jancborchardt** [comment]
What would be the problem if there are 200 entries in there? Sure it’s a long email, but at least you don’t have to log in.

Because currently for us the case is mostly that there are 1 or 2 entries (or only a few more), and for that it would be good to have the content directly in there:
![screenshot from 2018-09-25 13-12-24](https://user-images.githubusercontent.com/925062/46011166-527fd600-c0c5-11e8-8cf1-b5ca6e5b6cce.png)

**Comment by nickvergessen** [comment]
Subject is now part of the email again, as per nextcloud/announcementcenter#115
